core/src/main/java/greencity/exception/handler/CustomExceptionHandler.java

```diff
@@ -201,87 +108,24 @@ protected ResponseEntity<Object> handleMethodArgumentNotValid(MethodArgumentNotV
-     * Method interceptor exception {@link UnexistingUuidExeption}.
-     *
-     * @param ex      Exception which should be intercepted.
-     * @param request contain detail about occur exception.
-     * @return ResponseEntity which contain http status and body with message of
-     *         exception.
-     */
-    @ExceptionHandler({UnexistingUuidExeption.class, UnexistingOrderException.class})
-    public final ResponseEntity<Object> handleUuidNotFound(RuntimeException ex, WebRequest request) {
-        ExceptionResponce exceptionResponse = new ExceptionResponce(getErrorAttributes(request));
-        log.trace(ex.getMessage(), ex);
-        return ResponseEntity.status(HttpStatus.NOT_FOUND).body(exceptionResponse);
-    }
-
-    /**
-     * Method interceptor exception {@link NotFoundOrderAddressException}.
-     *
-     * @param ex         Exception which should be intercepted.
-     * @param webRequest contain detail about occur exception.
-     * @return ResponseEntity which contain http status and body with message of
-     *         exception.
-     */
-    @ExceptionHandler({NotFoundOrderAddressException.class})
-    public final ResponseEntity<Object> handleNotFoundOrderAddressException(NotFoundOrderAddressException ex,
-        WebRequest webRequest) {
-        ExceptionResponce exceptionResponce = new ExceptionResponce(getErrorAttributes(webRequest));
-        log.trace(ex.getMessage(), ex);
-        return ResponseEntity.status(HttpStatus.NOT_FOUND).body(exceptionResponce);
-    }
-
     /**
-     * Method interceptor exception {@link OrderNotFoundException}.
-     *
-     * @param ex         Exception which should be intercepted.
-     * @param webRequest contain detail about occur exception.
-     * @return ResponseEntity which contain http status and body with message of
-     *         exception.
-     */
-    @ExceptionHandler({OrderNotFoundException.class})
-    public final ResponseEntity<Object> handleOrderNotFoundException(OrderNotFoundException ex,
-        WebRequest webRequest) {
-        ExceptionResponce exceptionResponce = new ExceptionResponce(getErrorAttributes(webRequest));
-        log.trace(ex.getMessage(), ex);
-        return ResponseEntity.status(HttpStatus.NOT_FOUND).body(exceptionResponce);
-    }
-
-    /**
-     * Method interceptor exception {@link BadOrderStatusRequestException}.
+     * Method interceptor exception {@link SumOfOrderException}.
      *
      * @param ex         Exception which should be intercepted.
      * @param webRequest contain detail about occur exception.
      * @return ResponseEntity which contain http status and body with message of
      *         exception.
      */
-    @ExceptionHandler({BadOrderStatusRequestException.class})
-    public final ResponseEntity<Object> handleBadOrderStatusRequestException(BadOrderStatusRequestException ex,
+    @ExceptionHandler({SumOfOrderException.class, CourierLocationException.class,
+        LocationAlreadyCreatedException.class})
```

Comment:
This all exception has similar status as the previous, maybe it will be better to create one universal exception e.g. BadRequestException and handle it here?